src/main/java/io/github/thebusybiscuit/sensibletoolbox/util/STBUtil.java

```diff
@@ -723,4 +723,35 @@ public static boolean isPotionIngredient(Material type) {
         // TODO Fix Potion Ingredient lookup
         return false;
     }
+
+    /**
+     * Get the wall sign version of the sign material e.g. OAK_SIGN -> OAK_WALL_SIGN
+     *
+     * @param signType
+     *                 The material of the sign
+     * @return The wall sign version of that sign
+     */
+    @Nullable
+    public static Material getWallSign(@Nonnull Material signType) {
+        switch (signType) {
+            case OAK_SIGN:
+                return Material.OAK_WALL_SIGN;
+            case SPRUCE_SIGN:
+                return Material.SPRUCE_WALL_SIGN;
+            case BIRCH_SIGN:
+                return Material.BIRCH_WALL_SIGN;
+            case JUNGLE_SIGN:
+                return Material.JUNGLE_WALL_SIGN;
+            case ACACIA_SIGN:
+                return Material.ACACIA_WALL_SIGN;
+            case DARK_OAK_SIGN:
+                return Material.DARK_OAK_WALL_SIGN;
+            case CRIMSON_SIGN:
+                return Material.CRIMSON_WALL_SIGN;
+            case WARPED_SIGN:
+                return Material.WARPED_WALL_SIGN;
```

Comment:
Sensible Toolbox supports 1.14-1.16, so this might throw an exception on 1.14/1.15 here.
I can't really think of a really good way to approach this at the moment, so I'd suggest to use a string comparison for now and refactor it late unless you have a better idea how to account for this.